src/addRemove.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+import { toDoList } from './index.js';
+
+export class ToDoClass {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+ 
+  /* eslint-enable */
+
+  listCode() {
+    return `<li data-id="${this.index}"><div id="${this.index}" class="task"><input class="check-box list-${this.index}" type="checkbox"><p contenteditable="true">${this.description}</p></div>
+      <div><i class="fas fa-trash-alt icon-d"></i> <i class="fas fa-ellipsis-v icons"></i></div>`;
+  }
+
+  static addTo(newList) {
+    let id = 0;
+    if (toDoList.length > 0) {
+      id = toDoList[toDoList.length - 1].index + 1;
+    }
```

Comment:
- [ ] By default, new tasks should have the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Currently, your  indexes are 0-based, lest's fix that

![Screenshot from 2021-10-30 13-16-22](https://user-images.githubusercontent.com/40503704/139532520-98bda725-889a-4e40-9b55-9d969f85566f.png)